pkg/event/event.go

```diff
@@ -75,6 +77,8 @@ func New(obj interface{}, action string) Event {
 		component = string(object.Spec.Type)
 	case *api_v1.Secret:
 		kind = "secret"
+		reason = action
```

Comment:
@nabadger Is there a reason you're adding `reason` and `action` back into the case for Secret? These are set above, so this is unnecessary. Was there something that you saw that required it?

Reply:
@tylerauerbeck No sorry - it looks like my merge from upstream to resolve conflicts didn't remove those lines. 

I've fixed it now - thanks.